frontend/components/draggable/draggable.ts

```diff
@@ -138,7 +138,7 @@ export class Draggable extends EventTarget {
   /** Sets the position of the element. */
   setPosition(x: number, y: number) {
     let p = new Point(this.options.moveX ? x : 0, this.options.moveY ? y : 0);
-    if (this.bounds) p = p.clamp(this.bounds, this.options.margin ?? 0);
+    if (this.bounds && this.options.withinBounds) p = p.clamp(this.bounds, this.options.margin ?? 0);
```

Comment:
Do we need this? if `!options.withinBounds`, then `this.bounds` will never get set:
https://github.com/mathigon/studio/pull/21/files#diff-af89d6ee882fcf544c70ef15677b9dee0bd1d97ccdd890d5fd33ccc046cf700fL118

Reply:
In my testing it seemed like the die faces interactive was still getting clamping behavior without this change. I'll double check and report back.

Reply:
Alright looks like I was wrong, may have been a cache issue or something with my browser.